arelle/plugin/validate/ESEF/resources/authority-validations.json

```diff
@@ -0,0 +1,181 @@
+{
+    "copyright": [
+        "(c) 2022 Workiva, All rights reserved"
+    ],
+    "description": [
+        "Authority validation parameters.",
+        "",
+        "The format is as follows:",
+        "",
+        "A JSON object by authority with objects ",
+        "",
+        "reportPackageMaxMB: max file size in MB followed by \" zipped\" or \" unzipped\", or unlimited",
+        "ixTargetUsage: ok, warning or error",
+        "extensionAbstractContexts: ok, warning or error",
+        "G3.4.2: effective or ineffective when not applied by authority",
+        "G3.4.6: effective or ineffective when not applied by authority",
+        "additionalMandatoryTags: qnames of tags additionally required by authority when prefixed schema is imported"
+    ],
+    "default": {
+        "comment": "default parameters where not specified by authority",
+        "reportPackageMaxMB": "unlimited",
+        "identiferScheme": "http://standards.iso.org/iso/17442",
+	    "ixTargetUsage": "warning",
+        "extensionAbstractContexts": "warning",
+		"G3.4.2": "effective",
+		"G3.4.6": "effective",
+	    "outdatedTaxonomyURLs": [
+		    "http://www.esma.europa.eu/taxonomy/2017-03-31/esef_cor.xsd",
+		    "https://www.esma.europa.eu/taxonomy/2017-03-31/esef_cor.xsd"
+		],
+		"effectiveTaxonomyURLs": [
+		    "http://www.esma.europa.eu/taxonomy/2019-03-27/esef_cor.xsd",
+		    "https://www.esma.europa.eu/taxonomy/2019-03-27/esef_cor.xsd",
+		    "http://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor.xsd",
+		    "https://www.esma.europa.eu/taxonomy/2020-03-16/esef_cor.xsd",
+		    "http://www.esma.europa.eu/taxonomy/2021-03-24/esef_cor.xsd",
+		    "https://www.esma.europa.eu/taxonomy/2021-03-24/esef_cor.xsd"
+		]
+    },
+	"AT": {
+	    "name": "Austria",
+	    "reportPackageMaxMB": "100 zipped"
+	},
+	"BE": {
+	    "name": "Belgium",
+	    "reportPackageMaxMB": "50 zipped"
```

Comment:
Rather than parsing these by doing a split call, why not set the `zipped/unzipped/unlimited` as its own field?

Reply:
done